stl/inc/regex

```diff
@@ -4302,7 +4307,7 @@ void _Parser<_FwdIt, _Elem, _RxTraits>::_Quantifier() { // check for quantifier
             _Max = 1;
         } else if (_Mchar == _Meta_lbr) { // check for valid bracketed value
             _Next();
-            if (!_DecimalDigits()) {
+            if (!_DecimalDigits(regex_constants::error_badbrace)) {

```

Comment:
We should test for this as well (even if, yes, LLVM already does test this.)

Reply:
Thank you!
yes, LLVM tests only 4310 line (but their implementation passes our tests)
I added another test for 4320 line.

I also found that libstdc++ has some bugs with regex too: https://gcc.godbolt.org/z/evW3jfG3W
I will report them now.